survey/forms.py

```diff
@@ -164,18 +163,15 @@ def get_question_initial(self, question, data):
         if answer:
             # Initialize the field with values from the database if any
             if question.type == Question.SELECT_MULTIPLE:
-                initial = []
                 if answer.body == "[]":
-                    pass
-                elif "[" in answer.body and "]" in answer.body:
```

Comment:
@Pierre-Sassoulas I don't fully understand what the idea was here. It looks to me like a "]" and a "[" anywhere in the answer.body would cause this to be initiated. Maybe that's why you decided to slugify? To make sure there wouldn't be any "[" nor "]" in there?

Reply:
Possibly, I remember this was a bit hacky, we're serializing / deserializing a list of answers here.